### ISSUE TITLE
warning from sass

### ISSUE THREAD

**Issue by HairyRabbit** [csstools/postcss-sass#36]
```
`import sass from 'sass'` is deprecated.
Please use `import * as sass from 'sass'` instead.
```

would we update the source code?

**Comment by PedroCuadra** [comment]
https://github.com/csstools/postcss-sass/pull/38

I added a pull request about it. Hopefully they merge 🙏

**Comment by romainmenke** [comment]
This has been published as `5.1.0`.

Thank you for reporting
